### PR TITLE
Add 'ptf' to testbed config 

### PR DESCRIPTION
**Pull request by baiwei0427**
Signed-off-by: Wei Bai baiwei0427@gmail.com

### Description of PR

Summary: Add a new column "ptf" in testbed configuration file, which stands for the host name of the PTF instance. 

When we want to use AnsibleHost class to control a PTF instance, we must know its host name to get access information from the inventory file (e.g., https://github.com/Azure/sonic-mgmt/blob/master/ansible/veos.vtb)
 
Fixes # (issue)

### Type of change

- [] Bug fix
- [*] Testbed and Framework(new/improvement)
- [] Test case(new/improvement)

#### How did you do it?
Modify existing testbed configuration files. And modify testbed-cli.sh to parse the new configuration file format. 

#### How did you verify/test it?
Testbed

#### Any platform specific information?
N/A

#### Supported testbed topology if it's a new test case?
N/A

### CODE REVIEW

**Comment by baiwei0427** [comment]
> @baiwei0427 Why do we need this change? Also ptf_unknown

File: ansible/testbed-cli.sh [hunk 1]
```diff
@@ -81,10 +81,11 @@ function read_file
  testbed_name=${line_arr[1]}
  topo=${line_arr[2]}
  ptf_imagename=${line_arr[3]}
- ptf_ip=${line_arr[4]}
- server=${line_arr[5]}
- vm_base=${line_arr[6]}
- dut=${line_arr[7]}
+ ptf=${line_arr[4]}
```
Comment (msosyak) [inline ansible/testbed-cli.sh]: Could you move `ptf` column to the place before comment to be in sync with TestbedProcessing.py
https://github.com/Azure/sonic-mgmt/blob/27f225f05f56cac2e9c63fba517425a521ef1160/ansible/TestbedProcessing.py#L234

Comment (baiwei0427) [reply ansible/testbed-cli.sh]: @msosyak Thanks for point out this. It seems that we have found this problem but we put ptf column in different places :).  I think it is better to put PTF attributes (hostname, IP and image) together. What do you think of this? I can modify TestbedProcessing.py if necessary.

Comment (jleveque) [reply ansible/testbed-cli.sh]: @baiwei0427: I agree with your reasoning. I think you should modify TestbedProcessing.py and any other files to keep the order consistent. @msosyak: Can you please provide the names of all other files which should have their order modified to match here?

Comment (msosyak) [reply ansible/testbed-cli.sh]: 'ansible/TestbedProcessing.py' where we generate testbed.csv. 
In other places, testbed.csv is parsed by python and not sensitive for the order of column if I'm not missed something.

**Pull request merged**